phase-5/submissions/hashtags/RiccardoM

```diff
@@ -0,0 +1 @@
+5AA63243B254A92553F7DAA8D6283F0C6870DD67761AAD1BA1F897607C1495CB
```

Comment:
Hi @sun1534, do you mind removing this file? If you execute the following it should do it: 

```shell
rm phase-5/submissions/hashtags/RiccardoM
git add . 
git commit -m "Removed file" 
git push
```

Thanks!

Reply:
get done